### DISCUSSION TITLE
Is a capture-late possible with this package?

### DISCUSSION THREAD

**Discussion by AlbertoNoris** [flutter-stripe/flutter_stripe#1016] [Q&A]
Hi!
With our team we are trying to understand if the [Place a hold on a payment method](https://stripe.com/docs/payments/place-a-hold-on-a-payment-method) is available in flutter_stripe.
We've seen that we should turn the [CaptureMethod ](https://pub.dev/documentation/stripe_platform_interface/latest/stripe_platform_interface/CaptureMethod.html)to manual but then we are stuck on understading how we should capture it.
Before diving deep into trying to understand how to do it we were simply trying to understand ... is it possibile to do a capture later with this package and the payment sheet?
Thanks in advance!
Alberto :)

**Comment by remonh87** [comment]
hi, 

What I understood from the Stripe flow is that hold on payment is handled on the server. What you do  on the backend is tell stripe to authorize only. On the client nothing changes the backend needs to provide a payment intent that you can confirm using the payment sheet.
